### PR TITLE
Add Origin Trial documentation to the developer docs

### PR DESCRIPTION
**Pull request by ststimac**
Adding the origin trial docs to the developer doc site from @zouhir's explainer page.

### CODE REVIEW

File: microsoft-edge/origin-trials.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ms.author: msedgedevrel
+ms.date: 06/29/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: edge, web development, html, css, origin trials, developer
+---
+
+# Use Origin Trials in Microsoft Edge  
+
+Developers may use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you do not need to go to `edge://flags`and turn on feature flags.  For more information, see [experimental APIs][DeveloperMicrsoftEdgeOriginTrials].  Additionally, you may provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
```
Comment (Reezaali) [inline microsoft-edge/origin-trials.md:15]:
```suggestion
Developers may use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you do not need to go to `edge://flags` and turn on feature flags.  For more information, see [experimental APIs][DeveloperMicrsoftEdgeOriginTrials].  Additionally, you may provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.  
```

File: microsoft-edge/origin-trials.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    
+    > [!NOTE]
+    > To enroll a single or all subdomains, choose set the `Do you need to match all subdomains for the provided origin?` setting to `Yes`.  For example, `https://dev.contoso.com` is a single domain, and `https://*.contoso.com` uses a wildcard to represent all subdomains.  
+    
+    > [!IMPORTANT]
+    > The following origin formats are not allowed.  
+    > *   Specifying a subfolder on the origin.  
+    >     
+    >     ```https
+    >     https://contoso.com/path/subfolder
```
Comment (Reezaali) [inline microsoft-edge/origin-trials.md:45]: Should we remove this code block and replace with "...on the origin. For example..."?

File: microsoft-edge/origin-trials.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    >     ```  
+    > 
+    > *   Using an origin with query string parameters.  
+    >     
+    >     ```https
+    >     https://contoso.com/path/feature?test_id=12345
```
Comment (Reezaali) [inline microsoft-edge/origin-trials.md:51]: Similar comment as above.

**Comment by ststimac** [comment]
I'm good with @Reezaali's changes

File: microsoft-edge/origin-trials.md (hunk 2 above)
Comment (jm-trd-ms) [reply microsoft-edge/origin-trials.md]: Updated.

File: microsoft-edge/origin-trials.md (hunk 3 above)
Comment (jm-trd-ms) [reply microsoft-edge/origin-trials.md]: Updated.

**Comment by ststimac** [comment]
This looks good to me.

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**